sanic/app.py

```diff
@@ -205,7 +209,7 @@ def response(handler):
             if stream:
                 handler.is_stream = stream
 
-            routes = self.router.add(
+            new_routes = self.router.add(
```

Comment:
new_routes seems a little bit weird.. maybe we can just extend in place.

Reply:
Changed to inplace extend.